qlib/data/data.py

```diff
@@ -223,8 +223,11 @@ def convert_instruments(self, instrument):
             for _path in Path(C.get_data_path()).joinpath("instruments").glob("*.txt"):
```

Comment:
It is very strange that  `InstrumentProvider` has such such a function named `convert_instruments`